src/seeders/20210127161802-add-goals.js

```diff
@@ -0,0 +1,95 @@
+const now = new Date();
+
+const bulletedGoal = `
+  * Staff have become very Tech Savvy
+  * Grantee utilized down time to promote individual Professional Development and began training on Conscious Discipline to support staff and families with challenges.
+  * Created program changes to support COVID times such as:
+        Swivel cameras for observations
+        DOJO communication device for parents
+        Weekly ZOOM connects to support staff.
+`;
+
+const longGoal = `Grantee will receive support in developing a full enrollment action plan that covers the 12-month under-enrollment period.
+Grantee will receive strategies for ensuring they can consistently meet enrollment requirement via their ERSEA policies and procedures. Grantee will receive strategies for ensuring their partnership with Skyline leads to sustainable enrollment.`;
+
+const goals = [
+  {
+    id: 100,
```

Comment:
Is there a benefit to using explicit IDs here? Looks like the table has auto-increment setup.

Reply:
Yeah they are referenced below by the `grantGoals`

Reply:
In this case, should we use lower IDs and then bump the sequence above them, similar to how the user seeder works?

Reply:
In this case, should we use lower IDs and then bump the sequence above them, similar to how the user seeder works?